draft-ietf-webtrans-http2.md

```diff
@@ -781,6 +851,250 @@ Specification:
 
 : This document
 
+## Capsule Types
+
+The following entries are added to the "HTTP Capsule Types" registry established
+by {{HTTP-DATAGRAM}}:
+
+The `PADDING` capsule.
+
+Value:
+: 0x00
```

Comment:
!

Reply:
(Haven't generated any of them, as you'll see from the rest of the list that are all `0x00`)

Reply:
Updated